.env.example

```
@@ -1 +1,7 @@
-COINBASE_API_KEY=11111111111111111111111111111111111111111
\ No newline at end of file
+COINBASE_API_KEY=***************
+NEXT_PUBLIC_FIREBASE_PROJECT_ID=***************
+NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=***************
+FIREBASE_PRIVATE_KEY=*******************
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FIREBASE_PRIVATE_KEY key should go before the NEXT_PUBLIC_FIREBASE_PROJECT_ID key

(UnorderedKey)